ports/libopenmpt/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "libopenmpt",
-  "version": "0.6.7",
-  "description": "a cross-platform C++ and C library to decode tracked music files (modules) into a raw PCM audio stream.",
+  "version": "0.6.12",
```

```suggestion
  "version": "0.6.12",
```
Check the upstream and find that the latest version of libopenpmt is 0.7.3. Do you need to update to the latest version?

---

No. I don't.
Both versions are released on the same date. I only update the patch version this PR.
vcpkg's port uses a vendored buildsystem, and I don't want to deal with more changes.